src/main/java/yapper/ui/Ui.java

```diff
@@ -0,0 +1,98 @@
+package yapper.ui;
+
+import yapper.task.Task;
+import yapper.task.TaskList;
+import java.util.Scanner;
+import java.util.List;
+
+public class Ui {
+
+    private final Scanner scanner;
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void printWelcome() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm Yapper");
+        System.out.println(" What can I do for you Boss?");
+        System.out.println("____________________________________________________________");
+    }
+
+    public void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public String readCommand() {
+        return scanner.nextLine();
+    }
+
+    public void printError(String message) {
+        System.out.println("____________________________________________________________");
+        System.out.println(" " + message);
+        System.out.println("____________________________________________________________");
+    }
+
+    public void printLoadingError() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Error loading tasks from file!");
+        System.out.println("____________________________________________________________");
+    }
+
+    public void printGoodbye() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Bye Boss!. Yapper wants to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+
+    public void printTaskAdded(Task task, int taskCount) {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Got it Boss! I've added this task:");
+        System.out.println("   " + task);
+        System.out.println(" Okay Boss! Now you have " + taskCount + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public void printTaskRemoved(Task task, int taskCount) {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Okay bOSS ur task got removed");
+        System.out.println("   " + task);
+        System.out.println(" Okay Boss! Now you have " + taskCount + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
```

Comment:
There is inconsistency in the capitalization and formality of printed messages. The use of "bOSS" is inconsistent and unprofessional. It should be corrected to "Boss" for consistency and readability. The message "Okay bOSS ur task got removed" should be rephrased for professionalism and clarity. Consider using "Okay Boss, your task has been removed."

Reply:
Yes, this is very true